### ISSUE TITLE
about pretrain model

### ISSUE THREAD

**Issue by chenmyzju** [nqanh/affordance-net#11]
Hi
I found that I couldn't get a good model when I didn't use the pretrained model
I couldn't detect anything when I didn't use the pretrained model .
The same backbone network with VGG16 and after 50 thousand times iters.

so is it due to the hyperpameter ?
when I change the backbone network with the PVAlite network , I also find this issue 
thank you

**Comment by nqanh** [comment]
Yes, pretrained model helps us to reduce the training time and also better initialization for the model parameters. If you don't use it, then I guess you will need to train for a very long time, 50K inters may not be enough.
